Kernel/Storage/AHCIPort.h

```diff
@@ -83,6 +99,8 @@ class AHCIPort : public RefCounted<AHCIPort> {
     void start_request(AsyncBlockDeviceRequest&);
     void complete_current_request(AsyncDeviceRequest::RequestResult);
     bool access_device(AsyncBlockDeviceRequest::RequestType, u64 lba, u8 block_count);
+    Optional<size_t> calculate_descriptors_count(size_t block_count) const;
+    NO_DISCARD Optional<AsyncDeviceRequest::RequestResult> prepare_and_set_scatter_list(AsyncBlockDeviceRequest& request);
```

Comment:
Please use `[[nodiscard]]`, `NO_DISCARD` is a hack that is only used with `class` declarations to prevent `clang-format` from messing up the file. :)